package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rive-react-native",
-  "version": "4.0.4",
```

Comment:
I'd say this is worth a major version bump (seeing as iOS and Android bumped major). Or at least a minor, as we're adding new features.